src/app/pages/news/11.2/06-cron.md

```diff
@@ -1,6 +1,12 @@
 ## CRON expressions {#cron}
 
-Blah cron 
+You can now configure periodical jobs by using <a href="https://en.wikipedia.org/wiki/Cron">CRON</a> expressions.
+CRON expressions give you a wide spectrum of possible configurations. From once every minute up to once every day, week, month, or year. 
+
+- **User Synchronisation**: Define the time the user synchronization is executed with a CRON expression. 
```

Comment:
Maybe we want to emphasize, that there was a limitation in the previous version that is now gone?

User Synchronisation: Is now defined by a CRON expression, therefore now also allowing synchronizations multiple times a day or only on weekends.